### ISSUE TITLE
Enterolith in a fetal ultrasound

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11571]
**Preferred term label:**
Enterolith

**Synonyms**

**Definition (free text, please give PubMed ID)**
Hyperechoic (bright) foci in a dilated bowel

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
HP:0002023, ORPHA:887

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay 
Adding new term: 
- ID: HP:6001476
- label: Fetal enterolithiasis
- parents: [Abnormal fetal gastrointestinal system morphology (HP:0034207)](https://hpo.jax.org/app/browse/term/HP:0034207)
- pmid: PMID:30879001
- def: Enterolithiasis (concretions or stones forming within the gastrointestinal tract) is a prenatal sonographic sign defined by hyperechogenic foci within the - often distended - fetal bowel.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11571
- synonyms: Enterolith in a fetal ultrasound(NONE)
